examples/test_data/point_cloud_sample1.pts

```diff
@@ -0,0 +1,11 @@
+10
```

Comment:
Please merge https://github.com/intel-isl/Open3D/pull/3105 and add all test data to https://github.com/intel-isl/open3d_downloads

Reply:
Done